build/Settings.cs

```diff
@@ -52,9 +52,9 @@ public static class Settings
 
         public static string ExtensionBundleVersionRange = "[2.*, 3.0.0)";
 
-        public static string ExtensionBundleBuildVersion = "2.0.1";
+        public static string ExtensionBundleBuildVersion = "2.1.0";

```

Comment:
@soninaren - Ack on version? Minor version update instead of patch version?

Reply:
One of the assemblies (DurableTasks) was upgraded to a higher minor version, so I would expect this to also be a minor version upgrade. @soninaren, what do you think?